apps/web/src/modules/CourseSearch/hooks/useSearchCourseQueryParams/utils/extractSearchVarsFromQuery/index.ts

```diff
@@ -9,14 +9,16 @@ export function extractSearchVarsFromQuery(
   query: QueryParams,
   courseGroup: CourseGroup
 ): SearchCourseQueryVariables {
-  const { keyword, genEdTypes, dayOfWeeks, startTime, endTime } = query
+  const { keyword, genEdTypes, gradingTypes, dayOfWeeks, startTime, endTime } = query
 
   const genEdTypeArray = genEdTypes ? genEdTypes.split(',') : undefined
+  const gradingTypeArray = gradingTypes ? gradingTypes.split(',') : undefined
```

Comment:
im pretty sure we can use this instead
```suggestion
  const gradingTypeArray = gradingTypes?.split(',')
```

Reply:
Done, also updated other similar variables.